src/DataFrames.jl

```diff
@@ -17,7 +17,7 @@ using GZip
 using SortingAlgorithms
 using Base: Sort, Order
 
-@document
+#@document
```

Comment:
Are you sure this isn't needed by Documenter.jl? Same for `@comment` lines, which seem to correspond to headings below "API" at http://juliastats.github.io/DataFrames.jl/stable/.

@tshort Could you confirm?